config/opal_mca.m4

```diff
@@ -635,86 +634,16 @@ AC_DEFUN([MCA_CONFIGURE_M4_CONFIG_COMPONENT],[
     unset compile_mode
 ])
 
-
-######################################################################
-#
-# MCA_CONFIGURE_ALL_CONFIG_COMPONENTS
-#
-# configure all components in the given framework that have configure
-# scripts and should be configured according to the usual rules...
-#
-# USAGE:
-#   MCA_CONFIGURE_ALL_CONFIG_COMPONENTS(project_name,
-#                         framework_name,
-#                         all_components_variable,
-#                         static_components_variable,
-#                         dso_components_variable,
-#                         static_ltlibs_variable)
-#
-######################################################################
-AC_DEFUN([MCA_CONFIGURE_ALL_CONFIG_COMPONENTS],[
```

Comment:
Don't you need to also remove the invocation of this macro on line 459?

Reply:
yup; fixed.